target-platforms/packages/portal-7.4-ga1/build.js

```diff
@@ -84,9 +83,9 @@ function runBabel() {
 				'utf8'
 			);
 		}
-		catch (err) {
+		catch (error) {
```

Comment:
This one is wrong. There's already an `error` in scope.

Reply:
right! fixing...